REVIEWS.md

```diff
@@ -181,6 +205,13 @@ other version control systems. In the current version we removed the term from *
    think attempts to 'automatically update the documentation on code
    modification' are almost counter-productive -- availability of this
    level of documentation is not really what users crave.
+   
+> R/ We kept the current version of **Rule 5** about continues integration. The idea of the manuscript is to show 
+ the Git/GitHub infrastructure to bioinformatics community and general good practices. In general we don't aim to explain
+ in details best software engineering practices. However, **Rule 6** was extended commented in the supplmentary note explaining concepts like unit tests, 
```

Comment:
In general, we don't aim to provide detailed explanations of best software engineering practices.